### PR TITLE
ENG-1361 Fixes for user mgmt, permissions, labels

### PR DESCRIPTION
**Pull request by gidesan**
User without "user editing" permission cannot edit users
User without "user profile editing" permission cannot edit user profiles
User being not superuser permission cannot manage authorizations for users
User submenu items are pointing to AppBuilder screens (so, no more Admin Console involved with user management)
User submenu items are hidden if no enough permissions (instead of being present and pointing to a 403 page)
User list kebab menu actions properly hides unavailable ones if no enough permissions
“Save and edit profile” action in user edit form is properly translated
User screens breadcrumbs properly follow the new naming for user menu labels (User Management > Submenu Item ==> Users > Submenu Item)

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **gidesan**

If they are not already assigned, you can assign the PR to them by writing `/assign @gidesan` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/ui/internal-page/LegacyAdminConsoleMenuContainer.js
Comment (chalagashvili) [inline src/ui/internal-page/LegacyAdminConsoleMenuContainer.js:279]: I think users with following privileges can also view this menu item.
`CRUD_USERS_PERMISSION`
`VIEW_USERS_AND_PROFILES_PERMISSION`
`EDIT_USER_PROFILES_PERMISSION`

Comment (chalagashvili) [reply src/ui/internal-page/LegacyAdminConsoleMenuContainer.js:279]: If user has at least one of those 4 (including superuser priv) privileges, they should see list of users menu item

**Review by gReis89 — APPROVED** [review]
LGTM

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/862#pullrequestreview-505801113" title="Approved">gReis89</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [gReis89]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
